### PR TITLE
Update polish translations (to improve UX) & add missing translations

### PR DESCRIPTION
**Pull request by kniziol**
1. I updated polish translations to improve UX and make them more user-friendly.
2. Missing polish translations I found while comparing them to english translations.

🙃

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 composer.lock
 phpunit.xml
 vendor/
+.idea/
```
Comment (Devtronic) [inline .gitignore]: This has nothing to do with this bundle. I recommend to add the .idea directory to your local .git/info/exclude file

Comment (kniziol) [reply .gitignore]: Done (commit cd8b4a6)

**Comment by XWB** [comment]
@kniziol Please remove commit https://github.com/FriendsOfSymfony/FOSUserBundle/commit/f9bf3f04b61beb629e778bb0197efe77d87a0995 from this PR :)

**Comment by kniziol** [comment]
@XWB It's done

**Comment by XWB** [comment]
And merged, thanks :)

**Pull request merged**